docs/docs/30-how-to-guides/40-managing-projects.md

```diff
@@ -4,3 +4,39 @@ sidebar_label: Managing Projects
 ---
 
 # Managing Projects
+
+### Namespace Creation and Adoption in Kargo Projects
+
+When a new Kargo `Project` is created, it automatically generates a corresponding Kubernetes `namespace`.
+In scenarios where specific configuration requirements are needed, Kargo offers an adoption feature for
+pre-existing namespaces.
+
+Kargo can adopt namespaces that you create beforehand, if they are labeled
+with `kargo.akuity.io/project: "true"`.
+This allows you to pre-configure namespaces with necessary labels and resources.
```

Comment:
I think it would be good to add some commentary on why someone might do this. It's usually because their namespaces require specific labels or annotations _in order to comply with some policy enforced by a policy agent like Kyverno._